### PR TITLE
Adding Heroku buildpack

### PR DESCRIPTION
**Pull request by NipunaPrashan**
## Automation tests
N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,31 @@ Try WSO2 API Gateway with direct deployment on Heroku:
...
+5. Once that's done, you can deploy your app using this build pack any time by pushing to heroku:
+
+        git push heroku master
+        
+6. Finally, you need to scale your application with enough memory to run the API Gateway.
+        
+        heroku ps:scale web=1:standard-2x
 ## Documentation
 
 For more information about WSO2 API Gateway, see these WSO2 documentations:
```
Comment (sumedhassk) [inline README.md]: Shall we point to the correct doc here.?

File: bin/compile [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```
Comment (sumedhassk) [inline bin/compile]: Better to use #!/bin/sh since some one might not have bash

Comment (NipunaPrashan) [reply bin/compile]: This file is running in Heroku space and users doesn't have to run this file manually. Then we can keep this as it is.

File: README.md (hunk 1 above)
Comment (NipunaPrashan) [reply README.md]: Fixed with fa437c18c4260539479f906c919dc42ea4df2580

**Pull request merged**